database/src/main/java/net/tsystems/entitydao/PassengerDAO.java

```diff
@@ -0,0 +1,12 @@
+package net.tsystems.entitydao;
+
+import net.tsystems.entities.PassengerDO;
+
+import java.util.List;
+
+public interface PassengerDAO {
```

Comment:
You don't need all of those methods. The reason is that all of those methods are implementing into abstractDao

Reply:
Then is it supposed to have only "specific" methods, like getAll, getByName, etc.? Basically everything that's not simple CRUD.